clippy_lints/src/methods.rs

```diff
@@ -500,7 +528,12 @@ impl LateLintPass for Pass {
         match expr.node {
             hir::ExprMethodCall(name, _, ref args) => {
                 // Chain calls
-                if let Some(arglists) = method_chain_args(expr, &["unwrap"]) {
+                // GET_UNWRAP needs to be checked before general `UNWRAP` lints
+                if let Some(arglists) = method_chain_args(expr, &["get", "unwrap"]) {
+                lint_get_unwrap(cx, expr, arglists[0], false);
```

Comment:
Hmm it seems to me that you might want to panic with a message if the value isn't contained, in which case it seems like `.get().expect()` would be what you'd want to use, right?